.github/workflows/wheels.yml

```diff
@@ -0,0 +1,178 @@
+          CIBW_TEST_REQUIRES: hypothesis==6.52.1 pytest>=6.2.5 pytest-xdist pytest-asyncio>=0.17
+          CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: "python ci/fix_wheels.py {wheel} {dest_dir}"
+          CIBW_ARCHS_MACOS: x86_64 universal2
+
+      # Used to push the built wheels
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python[1] }}
+
+      - name: Test wheels (Windows 64-bit only)
+        if: ${{ matrix.buildplat[1] == 'win_amd64' }}
+        shell: cmd
+        run: |
+          python ci/test_wheels.py wheelhouse
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.python[0] }}-${{ startsWith(matrix.buildplat[1], 'macosx') && 'macosx' || matrix.buildplat[1] }}
+          path: ./wheelhouse/*.whl
+
+      - name: Upload wheels
+        if: success()
+        shell: bash
+        env:
+          PANDAS_STAGING_UPLOAD_TOKEN: ${{ secrets.PANDAS_STAGING_UPLOAD_TOKEN }}
+          PANDAS_NIGHTLY_UPLOAD_TOKEN: ${{ secrets.PANDAS_NIGHTLY_UPLOAD_TOKEN }}
+        run: |
+          source ci/upload_wheels.sh
+          set_upload_vars
+          # trigger an upload to
+          # https://anaconda.org/scipy-wheels-nightly/pandas
+          # for cron jobs or "Run workflow" (restricted to main branch).
+          # Tags will upload to
+          # https://anaconda.org/multibuild-wheels-staging/pandas
+          # The tokens were originally generated at anaconda.org
+          upload_wheels
+  build_sdist:
+    name: Build sdist
+    if: >-
+      github.event_name == 'schedule' ||
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'pull_request' &&
+      contains(github.event.pull_request.labels.*.name, 'Build')) ||
+      (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && ( ! endsWith(github.ref, 'dev0')))
+    runs-on: ubuntu-latest
+    env:
+      IS_PUSH: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
+      IS_SCHEDULE_DISPATCH: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+    steps:
+      - name: Checkout pandas
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          # versioneer.py requires the latest tag to be reachable. Here we
+          # fetch the complete history to get access to the tags.
+          # A shallow clone can work when the following issue is resolved:
+          # https://github.com/actions/checkout/issues/338
+          fetch-depth: 0
+      # Used to push the built wheels
+      - uses: actions/setup-python@v3
+        with:
+          # Build sdist on lowest supported Python
+          python-version: '3.8'
+      - name: Build sdist
+        run: |
+          pip install build
+          python -m build --sdist
+      - name: Test the sdist
+        run: |
+          # TODO: Don't run test suite, and instead build wheels from sdist

```

Comment:
Looks like this comment is no longer needed?

Reply:
I updated it to be more clear on what it entails. I removed the reference to the cibuildwheel PR since another one superseding it got merged(yay!).